processor/transformprocessor/README.md

```diff
@@ -29,8 +29,14 @@ Each condition and statement can access and transform telemetry using functions
 
 The transform processor allows configuring multiple context statements for traces, metrics, and logs.
 The value of `context` specifies which [OTTL Context](#contexts) to use when interpreting the associated statements.
-The conditions and statement strings, which must be OTTL compatible, will be passed to the OTTL and interpreted using the associated context. The conditions string should contain a string with a WHERE clause body without the `where` keyword at the beginning.
-Each context will be processed in the order specified and each condition and statement for a context will be executed in the order specified. Conditions are executed first, if a context doesn't meet the conditions, the associated statement will be skipped.
+The conditions and statement strings, which must be OTTL compatible, will be passed to the OTTL and interpreted using the associated context.
+The condition string should contain a string with a WHERE clause body without the `where` keyword at the beginning.
```

Comment:
```suggestion
The condition string should contain a Where clause body without the `where` keyword at the beginning.
```
* "string should contain a string" reads a little funny
* We've called them "Where" clauses below. Not sure whether to go with all caps or make them proper nouns, but usually we just go for proper noun, so that seems the most consistent.